llvm/test/CodeGen/AMDGPU/tgt-verify-llc-fail.ll

```diff
@@ -0,0 +1,6 @@
+; RUN: not not llc -mtriple=amdgcn -mcpu=gfx900 -verify-machineinstrs -enable-new-pm -verify-each -o - < %s 2>&1 | FileCheck %s
+
+define amdgpu_cs i32 @nonvoid_shader() {
+; CHECK: LLVM ERROR
+  ret i32 0
+}
```

Comment:
I think the generic IR verifier already gets this one. Also error messages need to be checked